File: routes/swift-to-cpp/certification/artifacts/module/source-runtime/004/equivalence_module.swift
func calculate(_ subtotal: Int64, _ tax: Int64) -> Int64 {
    if subtotal < 0 {
        return 0
    }
    return subtotal + tax
}

func clamp(_ value: Int64, _ minimum: Int64, _ maximum: Int64) -> Int64 {
    if value < minimum {
        return minimum
    }
    if value > maximum {
        return maximum
    }
    return value
}

func difference(_ left: Int64, _ right: Int64) -> Int64 {
    return left - right
}

func clampNumber(_ value: Double, _ minimum: Double, _ maximum: Double) -> Double {
    if value < minimum {
        return minimum
    }
    if value > maximum {
        return maximum
    }
    return value
}

func both(_ left: Bool, _ right: Bool) -> Bool {
    return left && right
}

File: .matrix223-python-let-candidate.xU9hBY/engines/polyglot-route-engine/fixtures/module/go/equivalence_module.go
package equivalencemodule

func calculate(subtotal int64, tax int64) int64 {
	if subtotal < 0 { return 0 }
	return subtotal + tax
}

func clamp(value int64, minimum int64, maximum int64) int64 {
	if value < minimum { return minimum }
	if value > maximum { return maximum }
	return value
}

func difference(left int64, right int64) int64 { return left - right }

func clampNumber(value float64, minimum float64, maximum float64) float64 {
	if value < minimum { return minimum }
	if value > maximum { return maximum }
	return value
}

func both(left bool, right bool) bool { return left && right }
